pino.js

```diff
@@ -245,12 +252,16 @@ Pino.prototype.asJson = function asJson (obj, msg, num) {
   return data + this.chindings + this.end
 }
 
+function getNoTime () {
+  return ''
+}
+
 function getTime () {
-  return Date.now()
+  return flatstr(',"time":' + Date.now())
 }
 
 function getSlowTime () {
-  return '"' + (new Date()).toISOString() + '"'
+  return flatstr(',"time":"' + (new Date()).toISOString() + '"')
```

Comment:
I updated it, but for some reason this comment isn't getting flagged as being on an out-of-date commit.